common/reviews/api/rush-lib.api.md

```diff
@@ -365,17 +371,26 @@ export interface ICredentialCacheOptions {
     supportEditing: boolean;
 }
 
+// @beta
+export interface ICustomTipInfo {
+    isMatch?: (str: string) => boolean;
+    // Warning: (ae-forgotten-export) The symbol "CustomTipSeverity" needs to be exported by the entry point index.d.ts
```

Comment:
`CustomTipSeverity` should get exported.

Reply:
exported